bingo/MultipleValues.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 
-from .Base.Chromosome import Chromosome
-from .Base.Mutation import Mutation
-from .Base.Crossover import Crossover
-from .Base.Generator import Generator
-from .Util.ArgumentValidation import argument_validation
+from bingo.Base.Chromosome import Chromosome
```

Comment:
lets stick with the explicit relative imports when we are in the bingo package  (the . imports)

we'll use full imports in the tests

please update your other files too

Reply:
Sounds good! Changes made :).